src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -151,6 +151,12 @@ module sonic-device_metadata {
                     }
                 }
 
+                leaf DualToRtype {
```

Comment:
Please add description

Reply:
If it's not dual tor device, what's the expected value? active-active?

Reply:
dualtortype is expected to be only `active-active` or `active-standby`